Integration/FRAIHMWORK Integrator Starter Kit Collection.json

```diff
@@ -0,0 +1,2038 @@
+												{
+													"key": "token",
+													"value": "{{access_token}}",
+													"type": "string"
+												}
+											]
+										},
+										"method": "POST",
+										"header": [],
+										"url": {
+											"raw": "{{baseUrl}}{{Monitoring}}/component/:uuid/refresh",
+											"host": [
+												"{{baseUrl}}{{Monitoring}}"
+											],
+											"path": [
+												"component",
+												":uuid",
+												"refresh"
+											],
+											"variable": [
+												{
+													"key": "uuid",
+													"value": "{{UUIDComponentResponse}}"
+												}
+											]
+										}
+									},
+									"response": []
+								}
+							]
+						}
+					]
+				}
+			]
+		},
+		{
+			"name": "Faults",
+			"item": [
+				{
+					"name": "POST",
+					"item": [
+						{
+							"name": "Posts a new fault that is affecting a component",
+							"event": [
+								{
+									"listen": "prerequest",
+									"script": {
+										"exec": [
+											""
+										],
+										"type": "text/javascript"
+									}
+								},
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.collectionVariables.set(\"UUIDFaultResponse\", \"null\")\r",
+											"var jsonData = pm.response.json();\r",
+											"pm.collectionVariables.set(\"UUIDFaultResponse\", jsonData.itemId);"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"auth": {
+									"type": "bearer",
+									"bearer": [
+										{
+											"key": "token",
+											"value": "{{access_token}}",
+											"type": "string"
+										}
+									]
+								},
+								"method": "POST",
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"name\": \"Fault of a component\",\n    \"code\": 0,\n    \"severity\": \"LOW\",\n    \"uuid\": \"{{$randomUUID}}\",\n    \"description\": \"Description of a fault\",\n    \"source\": \"{{UUIDComponentResponse}}\",\n    \"timeOfDetection\": \"2020-08-05T21:57:44.823Z\",\n    \"timeOfValidity\": \"2020-08-05T21:57:44.823Z\",\n    \"timeout\": 100,\n    \"extensions\": {\n        \"property\" :        {\n            \"name\": \"{{String}}\",\n            \"value\": \"{{String}}\",\n            \"type\": \"INTEGER\"\n        }\n    }\n}"
+								},
+								"url": {
+									"raw": "{{baseUrl}}{{Monitoring}}/fault",
+									"host": [
+										"{{baseUrl}}{{Monitoring}}"
+									],
+									"path": [
+										"fault"
+									]
+								},
+								"description": "Allows a component to report one or more faults currently affecting the component. Using this endpoint means that the client is not managing their own fault IDs, and the fault will require a new tracking UUID (that will be created and assigned in the response)."
```

Comment:
I think the documentation here is referencing the original API spec, and needs updating